src/share/classes/com/sun/btrace/runtime/instr/SynchronizedInstrumentor.java

```diff
@@ -74,6 +74,18 @@ public void visitInsn(int opcode) {
         } else if (opcode == MONITOREXIT) {
             onBeforeSyncExit();
         }
+        switch (opcode) {
```

Comment:
Yes, it would be correct in the exception situation, but if the method runs well, it would trigger onMethodReturn. So if ErrorReturnInstrumentor extends MethodReturnInstrumentor, it need to overwrite onMethodReturn by using empty method, is that right?